frontend/src/scenes/groups/Groups.tsx

```diff
@@ -36,12 +37,26 @@ export function Groups({ groupTypeIndex }: { groupTypeIndex: GroupTypeIndex }):
             setQuery={setQuery}
             context={{
                 refresh: 'blocking',
-                emptyStateHeading: 'No groups found',
+                emptyStateHeading: queryWasModified
+                    ? `No ${groupTypeNamePlural} found`
+                    : `No ${groupTypeNamePlural} exist because none have been identified`,
+                emptyStateDetail: queryWasModified ? (
+                    'Try changing the date range, or changing the property filters.'
+                ) : (
+                    <>
+                        Go to the{' '}
+                        <Link to="https://posthog.com/docs/product-analytics/group-analytics#how-to-create-groups">
+                            group analytics documentation
+                        </Link>{' '}
+                        to learn what needs to be done
```

Comment:
I'm not sure about this one because I'm not sure when this would be shown. "what needs to be done" is a bit vague. 
```suggestion
                        to learn how to create one
```